package.json

```diff
@@ -23,13 +23,13 @@
     "url": "https://github.com/CSNW/node-new-freshbooks/issues"
   },
   "homepage": "https://github.com/CSNW/node-new-freshbooks#readme",
-  "dependencies": {},
-  "devDependencies": {
-    "chai": "4.1.2",
-    "dotenv": "4.0.0",
-    "firebase": "4.12.1",
-    "mocha": "5.0.0",
-    "request": "2.83.0",
+  "dependencies": {
+    "chai": "^4.1.2",
+    "dotenv": "^4.0.0",
+    "firebase": "^4.12.1",
+    "firebase-admin": "^5.12.0",
+    "mocha": "^5.0.0",
```

Comment:
@schuttsm: Shouldn't chai & mocha be devDependencies & the rest be dependencies?